assets/panel/debugger.properties

```diff
@@ -648,6 +648,14 @@ watchExpressionsSeparatorLabel2=\u0020→
 # and its real name (if available).
 functionSearchSeparatorLabel=←
 
+# LOCALIZATION NOTE(gotoLineModal.placeholder): The placeholder
+# text displayed when the user searches for specific lines in a file
+gotoLineModal.placeholder=Go to line...
+
+# LOCALIZATION NOTE(symbolSearch.search.key2): The Key Shortcut for
```

Comment:
Please stop copying localization comments without updating them

Reply:
A more constructive way to say this would be "Please update this localization note to reflect the new key"

Reply:
It wasn't meant as an hostile comment towards the author, if it came across like that, I apologize.

For context: this is happening in several PRs every week, so I think there's a general issue to solve. Commenting strings is not mandatory, comments are there to give context to translators. On the other hand, I have the feeling most developers think they are, but copying and paste an existing comment is not the solution.

Reply:
It did come across like that, so thank you for apologizing. I have no knowledge of what other PR's look like (this is my very first one of any substance) or that the comments are optional, I'm just trying to match what's around the usage with the expectation of being corrected if necessary, so thanks for catching it, it was an oversight.